app-text/cmark/cmark-0.31.0.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{9..12} )
+
+inherit cmake python-any-r1
+
+DESCRIPTION="CommonMark parsing and rendering library and program in C"
+HOMEPAGE="https://github.com/commonmark/cmark"
+SRC_URI="https://github.com/commonmark/cmark/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD-2"
```

Comment:
I saw a MIT license in the source tarball, could you check and update this whenever you have time?